apps/ex_wire/lib/ex_wire/adapter/tcp/server.ex

```diff
@@ -104,15 +104,13 @@ defmodule ExWire.Adapter.TCP.Server do
     Logger.debug("[Network] Generating EIP8 Handshake for #{peer.host}")
 
     handshake =
-      %Handshake{remote_pub: peer.remote_id}
-      |> Handshake.initiate()
-
-    new_state = Map.merge(state, %{handshake: handshake})
+      Handshake.new(peer.remote_id)
```

Comment:
we can start pipeline operator with raw value

Reply:
Updated in 47eb9a8d5784216ef1b79409de580bfe1429e261